cdap-common/bin/functions.sh

```diff
@@ -904,6 +903,7 @@ cdap_sdk_start() {
       fi
     done
     echo
+    echo ${__pid} > ${__pidfile}
```

Comment:
should this be 4 lines further down?  if we ``die`` on line 908, do we want to leave the pidfile there?  If there's a reason for it, then 👍 lgtm, but i'm thinking maybe this should be moved to line 910